corehq/apps/hqadmin/static/hqadmin/js/admin_restore.js

```diff
@@ -1,5 +1,10 @@
 /* globals hqDefine ace */
```

Comment:
The `ace` global variable isn't going to be accessible in requirejs. You don't need the mode-xml or ext-searchbox dependencies, but you do need to add the main ace dependency to the dependency list and then the parameter list. [userreports/js/base.js](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/static/userreports/js/base.js) is an example of a requirejs module than uses ace.

Reply:
Thanks. updated the code use the ace with deps list